### PR TITLE
add xsd

### PR DESCRIPTION
**Pull request by lecorguille**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core

### CODE REVIEW

**Comment by lecorguille** [comment]
@bgruening Can I skip OSX? I really need this package to complete percolator one

**Comment by bgruening** [comment]
Sure, go on.

File: recipes/xsd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: xsd
+  version: 4.0.0+dep
```
Comment (bgruening) [inline recipes/xsd/meta.yaml]: can we change this version number, I'm not sure this is valid :(

Comment (lecorguille) [inline recipes/xsd/meta.yaml]: Done

**Pull request merged**